src/resources/License/License.ts

```diff
@@ -0,0 +1,18 @@
+import {SectionTypes} from '../Enums';
+import Resource from '../Resource';
+import {LicenseModel} from './LicenseInterfaces';
+
+export default class License extends Resource {
+    static baseUrl = '/rest/organizations';
+
+    get(organizationId: string, sectionName: SectionTypes) {
```

Comment:
What do you think of renaming `SectionTypes` to `LicenseSection`?
```suggestion
    get(organizationId: string, sectionName: LicenseSection) {
```

Reply:
Makes sense